numpy/polynomial/_polybase.py

```diff
@@ -479,6 +479,7 @@ def __setstate__(self, dict):
 
     def __call__(self, arg):
         off, scl = pu.mapparms(self.domain, self.window)
+        arg = np.asarray(arg)
         arg = off + scl*arg
```

Comment:
A simpler way to do the same thing might be to make sure `scl` is an array scalar rather than a regular scalar, since `array * list` already has the right behaviour of interpreting this list as an array.  One can do it here by writing this as `off + scl[...] * arg` - though it might make more sense to ensure `scl` is an array scalar inside `mapparms`.